hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -344,6 +344,8 @@ public Response get(
         };
         responseBuilder = Response
             .ok(output)
```

Comment:
This line should be removed, as it equals to setting `status = OK` and `entity = output`.

```suggestion
```

Reply:
done